principles.html

```diff
@@ -201,16 +201,22 @@ <h4 class="f_panelHead">Accessibility requirements related to text alternatives
     <ul class="listwithp">
       <li><a href="http://www.w3.org/WAI/WCAG20/quickref/#text-equiv" shape="rect">Guideline 1.1 - Text Alternatives</a></li>
     </ul>
-    <p class="listintro"><strong><acronym>UAAG</acronym> 1.0</strong></p>
-    <ul class="listwithp">
-      <li><a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-content-access" shape="rect">Guideline 2 - Ensure user access to all content</a></li>
-    </ul>
-    <p class="listintro"><strong><acronym>ATAG</acronym> 1.0</strong></p>
-    <ul class="listwithp">
-      <li><a href="http://www.w3.org/TR/ATAG10/#gl-access-support" shape="rect">Guideline 1 - Support accessible authoring practices</a></li>
-      <li><a href="http://www.w3.org/TR/ATAG10/#gl-prewritten-descs" shape="rect">Guideline 3 - Support the creation of accessible content</a></li>
-      <li><a href="http://www.w3.org/TR/ATAG10/#gl-make-accessible" shape="rect">Guideline 7 - Ensure that the authoring tool is accessible to authors with disabilities</a></li>
-    </ul>
+    <p class="listintro"><strong><acronym>UAAG</acronym> 2.0</strong></p>
+<ul class="listwithp">
+  <li>orginal: <a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-content-access" shape="rect">Guideline 2 - Ensure user access to all content</a></li>
+  <li>replaced 2 with: <a href="https://www.w3.org/TR/UAAG20/#principle-perceivable" shape="rect">Principle 1 - Ensure that the user interface and rendered content are perceivable</a></li>
+</ul>
+    <p class="listintro"><strong><acronym>ATAG</acronym> 2.0</strong></p>
+    <ul>
+      <li>original: <a href="http://www.w3.org/TR/ATAG10/#gl-access-support" shape="rect">Guideline 1 - Support accessible authoring practices </a></li>
+      <li>original: <a href="http://www.w3.org/TR/ATAG10/#gl-prewritten-descs" shape="rect">Guideline 3 - Support the creation of accessible content </a></li>
+      <li>original: <a href="http://www.w3.org/TR/ATAG10/#gl-make-accessible" shape="rect">Guideline 7 - Ensure that the authoring tool is accessible to authors with disabilities</a>    </li>
+    
+    
+      <li>replaced 1 &amp; 3 with: <a href="https://www.w3.org/TR/ATAG20/#part_b" shape="rect">Guideline B: Support the production of accessible content</a></li>
+      <li>Removed second item: <a href="http://www.w3.org/TR/ATAG10/#gl-prewritten-descs" shape="rect"><del>Guideline 3 - Support the creation of ac1cessible content</del></a></li>
+      <li>replaced 7 with: <a href="https://www.w3.org/TR/ATAG20/#part_a" shape="rect">Guideline A: Make the authoring tool user interface accessible</a></li>
+  </ul>
```

Comment:
Also here there is no need to keep the old bullets - just replace them please.

More importantly, also here please try to find the corresponding "Guidelines". Right now you are referencing the entire "Part A" and "Part B" of ATAG 2.0, which include *all* requirements. The reader is not really helped by that. The more specific guidelines here seem:
- Guideline A.2.1: (For the authoring tool user interface) Make alternative content available to authors.
- Guideline B.2.3: Assist authors with managing alternative content for non-text content.

There may be others too, I only took these from a cursory check of the table of contents for ATAG 2.0.